projects/melon-runtime/package.json

```diff
@@ -4,9 +4,11 @@
   "description": "Declarative modern .NET JavaScript runtime",
   "type": "module",
   "scripts": {
-      "prepublish": "dotnet publish Melon -o Output",
-      "go": "dotnet exec Output/Melon.dll",
-      "dev": "dotnet publish Melon -o Output && dotnet exec Output/Melon.dll"
+    "build": "dotnet publish Melon -o Output",
+    "exec": "dotnet exec Output/Melon.dll",
+    "dev": "dotnet watch --no-hot-reload --project Melon",
```

Comment:
`--no-hot-reload` doesn't block `Console.ReadLine()`

Note that .NET 7 has a new `--non-interactive` flag that fixes this with hot reload support, but we're using .NET 6 afaik. See https://github.com/dotnet/sdk/issues/23238#issuecomment-1008046943